recipes/blas/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set blas_pkg = "openblas" %}
+
+{% set version = "1.0.0" %}
+{% set build_number = 0 %}
+
+{% set version_post = (".post" + build_number|string) if build_number|int != 0 else "" %}
+{% set version_full = version + version_post %}
+
+
+package:
+  name: blas
+  version: {{ version_full }}
+
+build:
+  # Do not change the build number here.
+  number: 0
+  string: {{ blas_pkg }}
+
+requirements:
+  run:
+    - {{ blas_pkg }}
+
+test:
+  commands:
+    - conda list -n _test blas
```

Comment:
Setting the environment here feels like a horrific hack and quite fragile. Unfortunately, not setting it runs into this issue ( https://github.com/conda/conda-build/issues/910 ). So, not really sure there is another option ATM.